packages/postgres-database/src/simcore_postgres_database/migration/versions/cf3bac482ce0_add_file_meta_data_deprecation_date.py

```diff
@@ -0,0 +1,33 @@
+"""add services_meta_data deprecation date
+
+Revision ID: cf3bac482ce0
+Revises: 90ed05df20d7
+Create Date: 2022-06-21 14:43:57.915189+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "cf3bac482ce0"
+down_revision = "90ed05df20d7"
+branch_labels = None
+depends_on = None
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("services_meta_data", "deprecated")
+    # ### end Alembic commands ###
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "services_meta_data",
+        sa.Column(
+            "deprecated", postgresql.TIMESTAMP(), autoincrement=False, nullable=True
```

Comment:
ok I see this now for the first time, and I see @pcrespov used that already in the project snapshots. what is the advantage of timestamp over datetime?

Reply:
I cannot comment on pros/cons of str or datetime, sorry

Reply:
well then let's not mix them. can you check with @pcrespov why and if this should be used? since this is a copy paste of probably the only location where it was used... maybe let's check first if it has pros-cons

Reply:
from sqlalchemy [doc](https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.DateTime) I can see this brings timezone support. which in itself might be a good idea... but still we do not use this anywhere. Maybe you could create an issue that we could convert to that format all our times in the database that currently use the standard python datetime

Reply:
All good!`` postgresql.TIMESTAMP()`` is auto-generated from ``sa.DateTime`` by alembic and represents a special type for postgres

Reply:
sorry but I disagree.
looking at all of the other migration scripts show sa.DateTime in them, not a postgresql.TIMESTAMP(). So let's have alook together why that is (if it's something new from alembic versions or an error). If possible I would like that we do not start mixing up types more than necessary